tsconfig.json

```diff
@@ -4,25 +4,15 @@
     "module": "commonjs",
     "moduleResolution": "node",
     "noEmit": true,
-    "strictNullChecks": true,
+    "strict": true,
```

Comment:
Imo this is not a good idea because what's included in `strict` changes over time and will break the build in minor typescript updates.

Reply:
Usually these updates enhance type safety so this is a good thing?

Reply:
That's not the point, builds should be reproducible. I can't fix the build of a hundred repos when a new TS version comes out. The type safety enhancements almost always also trigger false positives.

Reply:
A better approach is to have an org-wide tsconfig as an npm package, and then a bot can do a PR to upgrade it. But that's a bit overkill here.